Dockerfile

```diff
@@ -22,4 +22,4 @@ ADD . /app
 
 EXPOSE 5000
 
-CMD flask run --host=0.0.0.0
+CMD flask run --host=0.0.0.0 && flask init-db
```

Comment:
this will init-db everytime the container is started?

Reply:
As i remember, yes.